### PR TITLE
Fixed -L shorthand bug for list command (#33)

### PR DESCRIPTION
**Pull request by devenes**
### What are you trying to accomplish?

Fixed -L shorthand bug for list command. (#33)

### What approach did you choose and why?

Added -L shorthand to use correctly with list command and added test for it. 

### Anything you want to highlight for special attention from reviewers?

Added test with using -L shorthand.

### CODE REVIEW

File: cmd/list_test.go [hunk 1]
```diff
@@ -62,6 +62,18 @@ func TestListWithIncorrectLimit(t *testing.T) {
 	assert.True(t, gock.IsDone(), internal.PrintPendingMocks(gock.Pending()))
 }
 
+func TestListWithIncorrectLimit2(t *testing.T) {
```
Comment (kotewar) [inline cmd/list_test.go:65]: Thanks for your PR, it looks almost good to me, just one small comment - 

Would you want to rename the test to something more meaningful as there's no difference in what this test and `TestListWithIncorrectLimit` are doing name and test wise. Feel free to call out something like, `TestListLimitShorthandUsingIncorrectLimit` that would tell the intention behind adding a similar test.

Comment (devenes) [reply cmd/list_test.go]: Thank you so much for your review. I renamed the function as you said.

**Comment by devenes** [comment]
I updated the function name, could you please review?

**Review by kotewar — APPROVED** [review]
Looks good to me!

**Pull request merged**